google/cloud/dns/zone.py

```diff
@@ -332,7 +332,7 @@ def list_resource_record_sets(self, max_results=None, page_token=None,
 
         :rtype: tuple, (list, str)
         :returns: list of
-                  :class:`google.cloud.dns.resource_record_set.ResourceRecordSet`,
+                  :class:`~.resource_record_set.ResourceRecordSet`,
```

Comment:
That's fine with me as well. (I made sure to make it also be a relative path per our previous discussion.)